examples/rainbowkit-demo/package.json

```diff
@@ -10,17 +10,20 @@
     "@chakra-ui/react": "^2.2.1",
     "@coinbase/wallet-sdk": "^3.2.0",
     "@rainbow-me/rainbowkit": "^0.8.1",
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.3.0",
-    "@testing-library/user-event": "^13.5.0",
     "ethers": "^5.7.2",
     "react": "^18.1.0",
-    "react-app-rewired": "^2.2.1",
     "react-dom": "^18.1.0",
-    "react-scripts": "5.0.1",
     "util": "^0.12.4",
     "wagmi": "^0.9.6"
   },
+  "devDependencies": {
+    "react-app-rewired": "^2.2.1"
+  },
+  "overrides": {
+    "react-scripts": {
+      "nth-check": "^2.0.1"
```

Comment:
to address dependabot alerts 
![image](https://user-images.githubusercontent.com/966662/211961561-3d663891-3df3-47c8-ae2b-8cbd8344c066.png)
https://github.com/facebook/create-react-app/issues/12132#issuecomment-1130249584